<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvailableItemWindow</class>
 <widget class="QDialog" name="AvailableItemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="contentList">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>341</width>
     <height>401</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="selectionRectVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="contentBackButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Main Menu</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="metadataView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>281</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addToCartButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to Cart</string>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { color : red; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
